app/assets/javascripts/u2f_registrations.js

```diff
@@ -0,0 +1,39 @@
+;(function() {
+  'use strict';
+
+  /*
+   * Register a u2f device
+   */
+  function registerU2fDevice(formElement, responseInput) {
+    let appId = formElement.querySelector('[name=u2f_app_id]').value;
+    let registrationRequests = JSON.parse(formElement.querySelector('[name=u2f_registration_requests]').value);
+    let registeredKeys = JSON.parse(formElement.querySelector('[name=u2f_registered_keys]').value);
```

Comment:
Call these `sign_requests` this is what they are for authentication (they are the same thing)